magi-p3-post.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  files: [
+    'package.json',
+    'test/accessibility-test.html',
+    'test/accessibility-test.html',
```

Comment:
I guess one of these identical entries could be removed?

Reply:
Thanks, removed.